src/components/elements/RelativeDateDisplay.tsx

```diff
@@ -73,12 +73,13 @@ const RelativeDateDisplay = ({
           ...TypographyProps.sx,
         }}
       >
+        {prefixVerb && <>{prefixVerb} </>}
+        {formattedDateRelative}
+        {suffixText && <> {suffixText}</>}
         {/* Include the tooltip text as visually hidden for accessibility */}
-        {prefixVerb || null} {formattedDateRelative}{' '}
         <Box sx={customVisuallyHidden}>
           ({formattedDate} {tooltipSuffixText})
-        </Box>{' '}
-        {suffixText || null}
+        </Box>
```

Comment:
Updates here are to avoid a space after the date, such as "Last updated today ."

Reply:
This spacing removal is impacting other locations where we expect, needs thorough QA before going out:
<img width="268" alt="Screenshot 2025-07-01 at 3 05 35 PM" src="https://github.com/user-attachments/assets/34bcfd29-466f-45d7-b954-b0c38cf4be18" />